http/cves/2024/CVE-2024-29059.yaml

```
@@ -20,6 +20,7 @@ info:
     epss-percentile: 0.99857
     cpe: cpe:2.3:a:microsoft:.net_framework:*:*:*:*:*:*:*:*
   metadata:
+    runzero-match: service["http.head.server"] matches "ms .net remoting"
```

```suggestion
    runzero-match: service["http.head.server"] matches "(?i)ms .net remoting"
```
looks like this could benefit from a case-insensitive match

---

Good catch, fixed. Thank you.